runtime-util/src/main/java/io/camunda/connector/runtime/util/outbound/OutboundConnectorValue.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.runtime.util.outbound;
+
+import io.camunda.connector.api.outbound.OutboundConnectorFunction;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class OutboundConnectorValue {
```

Comment:
I find that name a bit odd and generic, to be honest. Any thoughts on `OutboundConnectorConfiguration` maybe?
```suggestion
public class OutboundConnectorConfiguration {
```

Reply:
Don't have a strong feeling on this. "Value" is how those things are named in spring-zeebe - but I also like the one you propose. Happy to adjust it.